lldb/include/lldb/Core/SourceManager.h

```diff
@@ -74,6 +74,10 @@ class SourceManager {
 
     const Checksum &GetChecksum() const { return m_checksum; }
 
+    llvm::once_flag &GetChecksumWarningOnceFlag() {
```

Comment:
This getter seems rather pointless, isn't the only user in SourceManager itself?

Reply:
Yes, but this isn't SourceManager.